### PR TITLE
Fixed WCAG issues.

### PR DESCRIPTION
**Pull request by ahamed**
This Pull Request fixes the issues #50, #51, #53, #54, #56. Those all are WCAG issues.

This PR changes -

- Administrator login panel's glossy Joomla! logo by flat colored Joomla! logo.
- Contrast Ratio issue for installation step's **hints** texts and **login** button of the administrator login page.
- Button's **discernible text** issues.

### CODE REVIEW

File: build/media_source/com_cpanel/js/admin-cpanel-default.es6.js [hunk 1]
```diff
@@ -49,6 +49,52 @@
...
+   * @since   4.0.0
+   */
+  const toggleCard = (card) => {
+    card.addEventListener('click', (event) => {
+      event.preventDefault();
+      const targetBody = document.getElementById(card.getAttribute('data-target'));
+      const { currentTarget: el } = event;
+
+      if (el) {
+        // Toggle aria-expanded arrtibute
```
Comment (brianteeman) [inline build/media_source/com_cpanel/js/admin-cpanel-default.es6.js]:
```suggestion
        // Toggle aria-expanded attribute
```

**Comment by brianteeman** [comment]
Several of the changes here break the usage of the atum template. This template is supposed to be an alternative template and not a replacement template. Everytime you do something to break atum you create more work later on.

**Comment by brianteeman** [comment]
Can you **please** close this pull request and submit one pull request per issue (unless obviously one change solves multiple issues). This makes it much easier to test and to track changes. It also makes it much quicker to test and merge as an error in one part does not prevent the other parts being merged.

**Comment by ahamed** [comment]
> Can you **please** close this pull request and submit one pull request per issue (unless obviously one change solves multiple issues). This makes it much easier to test and to track changes. It also makes it much quicker to test and merge as an error in one part does not prevent the other parts being merged.

Those are small issues and are similar too. The merge could be postponed until all of them are fixed. 
And next PRs shall be individual for every issue.

**Comment by brianteeman** [comment]
It is bad practice and should be avoided.

https://medium.com/@fagnerbrack/one-commit-one-change-3d10b10cebbf

https://victoria.dev/blog/git-commit-practices-your-future-self-will-thank-you-for/

**Pull request closed without merging**